webjump/magento2-module-braspagpagador
# Added Cielo30 as payment provider

### Description (*)

Braspag created a new provider to work with SOAP integration for Cielo credit card called Cielo30. The implementation added this provider to available payment list.

### Manual testing scenarios (*)

1. Open Magento admin store and check if Cielo30 payments information are available on provider config for credit card and debit card

### Contribution checklist (*)
